apps/settings/serializers/auth/ldap_ha.py

```diff
@@ -84,11 +86,12 @@ class LDAPHASettingSerializer(serializers.Serializer):
 
     AUTH_LDAP_HA = serializers.BooleanField(required=False, label=_('LDAP HA'))
     AUTH_LDAP_HA_SYNC_ORG_IDS = OrgListField()
+    
+    periodic_key = 'AUTH_LDAP_HA_SYNC_IS_PERIODIC'
+    interval_key = 'AUTH_LDAP_HA_SYNC_INTERVAL'
+    crontab_key = 'AUTH_LDAP_HA_SYNC_CRONTAB'
 
-    def post_save(self):
-        keys = ['AUTH_LDAP_HA_SYNC_IS_PERIODIC', 'AUTH_LDAP_HA_SYNC_INTERVAL', 'AUTH_LDAP_HA_SYNC_CRONTAB']
-        kwargs = {k: self.validated_data[k] for k in keys if k in self.validated_data}
-        if not kwargs:
-            return
+    @staticmethod
+    def import_task_function(**kwargs):
         from settings.tasks import import_ldap_ha_user_periodic
         import_ldap_ha_user_periodic(**kwargs)
```

Comment:
Based on the above provided code snippet, there seem to be no issues, irregularities, and optimizations that could be suggested based solely on the contents of this file. If you need more comprehensive code review, please provide me with specific lines or comments within those blocks of code.